### ISSUE TITLE
[nextage_gazebo] World is very dark on melodic

### ISSUE THREAD

**Issue by pazeshun** [tork-a/rtmros_nextage#370]
On melodic:
```
$ roslaunch nextage_gazebo nextage_world.launch
```
![Screenshot from 2020-04-17 18-48-43](https://user-images.githubusercontent.com/14994939/79556612-4bbc1800-80dc-11ea-8f02-eeca89c82afa.png)

On kinetic:
```
$ roslaunch nextage_gazebo nextage_world.launch
```
![Screenshot from 2020-04-17 18-48-06](https://user-images.githubusercontent.com/14994939/79556670-5c6c8e00-80dc-11ea-83c7-27f8e2669f97.png)

**Comment by pazeshun** [comment]
When I added many lights:
![Screenshot from 2020-04-17 18-58-19](https://user-images.githubusercontent.com/14994939/79557409-78246400-80dd-11ea-93be-6264bf317e60.png)

**Comment by k-okada** [comment]
yeah, the world is dark, but the object looks good, and the PointClouds seems ok

I first thought this is because I prefer super heavy "mobile workstation" (with GPU) and @pazeshun likes thin and light laptop (without GPU?)... Partially it is true. The robot itself seems dark on my GPU machine and bright on T480s. And we do not have shadows on T480s, But the color of objects in Gazebo and PointClouds seems ok and both machines run at ~60Hz.

T480s (see glx log https://gist.github.com/k-okada/ff01c0a2755e5d35e21612c571c283a4 )
![Screenshot from 2020-04-17 20-07-48](https://user-images.githubusercontent.com/493276/79563145-2bde2180-80e7-11ea-86d7-5e8de70cce46.png)



P52s
![Screenshot from 2020-04-17 19-41-07](https://user-images.githubusercontent.com/493276/79561235-f97ef500-80e3-11ea-9a6a-08df72539b0f.png)

**Comment by k-okada** [comment]
And it is always better to check if the robot has collade file. It is very hard to write a program that correctly interprets the model file, because the collad file is so flexible that there are many way of describing the model. So if we convert the .dae to .stl , we will get something like this;

T480s:
![Screenshot from 2020-04-17 19-59-02](https://user-images.githubusercontent.com/493276/79562699-698e7a80-80e6-11ea-843a-13ba5a0b37a3.png)

Please use this script to reproduce this https://gist.github.com/k-okada/057f391d7fcab4a0ebb1bd3a182f839f

**Comment by pazeshun** [comment]
I confirmed [your tutorial](https://github.com/start-jsk/rtmros_tutorials/pull/572) works and color of object can be acuquired from point cloud (I also tried blue box and red box, there seemed no problem).
However, I think light issue should be fixed, because Nextage itself seems black even in RGB image.
![Screenshot from 2020-04-17 20-25-32](https://user-images.githubusercontent.com/14994939/79564628-fdae1100-80e9-11ea-997b-2a6f42c3dc52.png)

**Comment by k-okada** [comment]
I see ,from https://bitbucket.org/osrf/gazebo/issues/2623/no-shadows-and-sun-light-with-non-nvidia, what we can do is to disable shadows ...

see
https://github.com/k-okada/rtmros_tutorials/commits/disable_shadow
for 'sample.world' settings for disabling shadows.

![Screenshot from 2020-04-17 20-45-04](https://user-images.githubusercontent.com/493276/79565991-bd03c700-80ec-11ea-8b53-10bd5fab9b2f.png)

**Comment by pazeshun** [comment]
Wow, https://github.com/k-okada/rtmros_tutorials/commit/659d630a27d34230ea3aa906d0297e3d57140bec makes the dawn!!!
![Screenshot from 2020-04-17 23-00-50](https://user-images.githubusercontent.com/14994939/79577498-c3e80500-80ff-11ea-8ce7-d352235cc90f.png)
My PC is T460s including NVIDIA GPU, but I don't activate it.

**Comment by the-raspberry-pi-guy** [comment]
Hi! Working on something similar here - how did you get the Nextage Open model in melodic? All of the docs point towards older versions of ROS and I am struggling here! Any advice appreciated (relatively new user here) :)

**Comment by pazeshun** [comment]
@the-raspberry-pi-guy 
Sorry for late, and I feel you can work Nextage now according to #373 .
But I'm writing information just in case.

Now our rtmros base software is released on [ROS testing repository](http://wiki.ros.org/TestingRepository).
If you already use that repository, what you should have to build from source is [rtmros_hironx](https://github.com/start-jsk/rtmros_hironx) and rtmros_nextage.
All other dependencies should be installed via `rosdep install` to those repositories.
Currently, #365 is required to build rtmros_nextage on melodic.
If you don't want to use ROS testing repository and cannot wait about two weeks (delay to find mistakes), you have to build [hrpsys](https://github.com/fkanehiro/hrpsys-base), [rtmros_common](https://github.com/start-jsk/rtmros_common), rtmros_hironx, and rtmros_nextage at least.

**Comment by the-raspberry-pi-guy** [comment]
Hi @pazeshun, thanks for your response :) I did indeed manage to get the Nextage Open model to load in Gazebo - thank you for your advice with this.

Do you have any advice for issue #373? The model seems to be not calibrated properly - I am getting a lot of vibration when trying out movements.
